pkg/tanka/prune.go

```diff
@@ -42,14 +44,20 @@ func Prune(baseDir string, opts PruneOpts) error {
 		return nil
 	}
 
-	// print diff
-	diff, err := kubernetes.StaticDiffer(false)(orphaned)
-	if err != nil {
-		// static diff can't fail normally, so unlike in apply, this is fatal
-		// here
-		return err
+	if opts.Summarize {
+		for _, m := range orphaned {
+			fmt.Printf("Pruning %s\n", m.KindName())
+		}
```

Comment:
This is a really awesome idea, and the implementation looks really good as well. Good job!

Our `diff` command already has a similar flag which prints a histogram-like summary based on `diffstat(1)`:
https://github.com/grafana/tanka/blob/4a96a91c7e70f75ddde5685a66794583ac001300/pkg/kubernetes/diff.go#L67-L69

For consistency, I think this would make sense here as well. Even though it would show a lot of deleted files, I feel this plays well with our other diff outputs.

wdyt?